modules/packages/TOML.chpl

```diff
@@ -748,7 +748,7 @@ Used to recursively hold tables and respective values
           final += ']';
           return final;
         }
-        when fieldReal do return val.re;
+        when fieldReal do return val.re:string;
```

Comment:
If your change made these casts necessary, I think that should be mentioned in the PR message

Reply:
It did. Basically the function was doing
``` chapel
proc f() : string {
  var x:int = 1;
  return x;
}
```
and the compiler was nicely coercing that, even though such a coercion is not normally allowed. I.e. it's a type error. I'll note it in the PR message.